content/v2.0/security/https_setup.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Enabling HTTPS with InfluxDB
+description: >
+  Enable HTTPS and Transport Security Layer (TLS) secure communication between clients and your InfluxDB servers.
+weight: 201
+menu:
+  v2_0:
+    name: Enabling HTTPS
+    parent: Security & authorization
+v2.0/tags: [security, authentication, tls, https]
+---
+
+Enabling HTTPS encrypts the communication between clients and the InfluxDB server.
+When configured with a signed certificate, HTTPS can also verify the authenticity of the InfluxDB server to connecting clients.
+
+{{% warn %}}
+InfluxData [strongly recommends](/influxdb/v1.7/administration/security/) enabling HTTPS, especially if you plan on sending requests to InfluxDB over a network.
+{{% /warn %}}
+
+## Requirements
+
+To enable HTTPS with InfluxDB, you'll need an existing or new InfluxDB instance
+and a Transport Layer Security (TLS) certificate (also known as a Secured Sockets Layer (SSL) certificate).
```

Comment:
- I think we can leave out needing an InfluxDB instances. That can safely be assumed.
- Don't use future tense when stating requirements: ~...you'll need...~ -> you need